includes/dintero-checkout-functions.php

```diff
@@ -103,10 +103,6 @@ function dintero_update_wc_shipping( $data ) {
  * @return void
  */
 function dintero_confirm_order( $order ) {
-	if ( ! empty( $order->get_date_paid() ) ) {
```

Comment:
Why is this removed @montazarr. What would happen if a customer triggers this function multiple times?

Reply:
The date paid is only set when the order is processing or completed. Since orders that are created on callback is pending payment, they don't have a date paid, which prevents the order from being processed. This, I suggest we have to remove this check.

I don't think the would have issue with orders being processed multiple times since we already have checks for that in the order management class.

Reply:
Ok in that case all good!